Upstream-Status="Inappropriate [embedded specific]"

Take out -I /usr/include/slang from CFLAGS.  
It is not safe for cross compiling and gives this error with gcc 4.6.0
|     CC perf.o
| cc1: error: include location "/usr/include/slang" is unsafe for cross-compilation [-Werror=poison-system-directories]
| cc1: all warnings being treated as errors

Nitin A Kamble <nitin.a.kamble@intel.com> 2011/04/21

diff --git a/tools/perf/Makefile b/tools/perf/Makefile
index bc35f19..c74e445 100644
--- a/tools/perf/Makefile
+++ b/tools/perf/Makefile
@@ -569,7 +569,7 @@ else
 		BASIC_CFLAGS += -DNO_NEWT_SUPPORT
 	else
 		# Fedora has /usr/include/slang/slang.h, but ubuntu /usr/include/slang.h
-		BASIC_CFLAGS += -I/usr/include/slang
+#		BASIC_CFLAGS += -I/usr/include/slang
 		EXTLIBS += -lnewt -lslang
 		LIB_OBJS += $(OUTPUT)util/ui/setup.o
 		LIB_OBJS += $(OUTPUT)util/ui/browser.o
